setup.py

```diff
@@ -15,9 +15,10 @@
         ('share/' + package_name, ['package.xml']),
     ],
     install_requires=['setuptools'],
-    maintainer='Austin Hendrix',
-    maintainer_email='namniart@gmail.com',
-    author='Graylin Trevor Jay',
+    maintainer='Chris Lalancette',
+    maintainer_email='clalancette@openrobotics.org',
+    author='Graylin Trevor Jay, Austin Hendrix',
+    author_email='namniart@gmail.com',
```

Comment:
not sure if setup.py likes a different number of author and author_email. Id suggest to ditch the email field as there is no way to know which author it belongs to